ando/engine.py

```diff
@@ -198,10 +238,26 @@ def is_AnDO_verbose_Format(names):
     Main method of the validator. uses other class methods for checking
     different aspects of the directory path.
 
-    :param names: list of names founds in the path
+    Args:
+        names ([list]): [names to check]
+
+    Raises:
+        ExperimentError: raised if it doesn't the experiment rules
```

Comment:
replace "doesn't" by "does not", here and everywhere...

("doesn't" is only oral langage; in written english, we should use "does not")

Reply:
also, the verb is missing in this sentence... "does not RESPECT the experiment rule"... correct everywhere also ;)

Reply:
Done